Sources/BraintreeDropIn/Components/BTUIKCardListLabel.m

```diff
@@ -53,25 +53,28 @@ - (void)setAvailablePaymentOptions:(NSArray *)availablePaymentOptions {
 - (void)updateAppearance {
     NSMutableAttributedString *at = [[NSMutableAttributedString alloc] initWithString:@""];
     NSMutableArray *attachments = [NSMutableArray new];
-    BTUIKPaymentOptionCardView *hint = [BTUIKPaymentOptionCardView new];
-    hint.frame = CGRectMake(0, 0, [BTUIKAppearance smallIconWidth], [BTUIKAppearance smallIconHeight]);
-    hint.borderColor = UIColor.systemGrayColor;
-    
+    NSString *accessibilityLabel = [NSString stringWithFormat:@"%@: ", BTDropInLocalizedString(CARD_ICONS_LABEL)];
+    BTUIKPaymentOptionCardView *cardIconView = [BTUIKPaymentOptionCardView new];
+    cardIconView.frame = CGRectMake(0, 0, [BTUIKAppearance smallIconWidth], [BTUIKAppearance smallIconHeight]);
+    cardIconView.borderColor = UIColor.systemGrayColor;
+
     for (NSUInteger i = 0; i < self.availablePaymentOptions.count; i++) {
         NSTextAttachment *composeAttachment = [NSTextAttachment new];
         BTDropInPaymentMethodType paymentOption = ((NSNumber*)self.availablePaymentOptions[i]).intValue;
-        composeAttachment.accessibilityLabel = [BTUIKViewUtil nameForPaymentMethodType:paymentOption];
-        hint.paymentMethodType = paymentOption;
-        [hint setNeedsLayout];
-        [hint layoutIfNeeded];
-        UIImage *composeImage = [self imageWithView:hint];
+        accessibilityLabel = [@[accessibilityLabel, [BTUIKViewUtil nameForPaymentMethodType:paymentOption]] componentsJoinedByString:@","];
+        
+        cardIconView.paymentMethodType = paymentOption;
+        [cardIconView setNeedsLayout];
```

Comment:
nit: I can't 100% recall but I think since we're calling `layoutIfNeeded` directly `setNeedsLayout` may result in a no-op.

Reply:
Hm, based on the docs difference b/w the two functions yeh it would seem like we could get rid of `setNeedsLayout`, however it is breaking the UI surprisingly. Just going to leave this as-is.